### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by guerreirobeatriz**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -13,9 +19,9 @@ The Store Link app provides blocks responsible for displaying links in other the
 
 1. Add `store-link` app to your theme's dependencies in the `manifest.json`, for example:
 
-```jsonc
-{
-  "vtex.store-link": "0.x"
+```diff
+  "dependencies": {
++   "vtex.store-link": "0.x"
 }
```
Comment (kaisermann) [inline docs/README.md]:
```suggestion
  }
```

Comment (klzns) [inline docs/README.md]: 👀 
```suggestion
 }
```

**Review by kaisermann — APPROVED** [review]

File: docs/README.md [hunk 2]
```diff
@@ -110,24 +92,27 @@ When creating a Link URL you have the query string values available. Example:
...
+| `category3`  | Third hieghest level category.                |
+| `category4`  | Fourth highest level category.                 |
+| `department` | Product department.                            |
+| `productId`  | Product ID.                                    |
+| `skuId`      | Current selected SKU ID.                       |
+| `slug`       | The link text used to create the product link. |
+
+Using one of these variables, you will be able to structure any desired URL for your store, such as a link to a given product department (`/{department}`).
+
+To build URLs with variables related to the product specifications, you should use the following format: `{specificationGroups.|{{groupName}}|.specifications.|{{specificationName}}|}`, replacing the curly brackets according to your scenario. For example:
```
Comment (klzns) [inline docs/README.md]: It's a bit confusing. Maybe this?
```suggestion
To build URLs with variables related to the product specifications, you should use the following format: `{specificationGroups.GROUP NAME HERE.specifications.SPECIFICATION NAME HERE}`, replacing the curly brackets according to your scenario. For example:
```

File: docs/README.md [hunk 3]
```diff
@@ -139,7 +124,8 @@ You will also be able access values related to the product specifications throug
   }
 }
 ```
-In this case, `App Data` is the specificationGroup name and `vtex-url` is the specification name.
+
+In the example above, `App Data` is the specification group name and `vtex-url` is the product specification name.
```
Comment (klzns) [inline docs/README.md]: I would change the above example to a more common one:

`{specificationGroups.Design.specifications.Dimensions}`

**Review by klzns — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 2 above)
Comment (kaisermann) [reply docs/README.md]: Maybe `[GROUP_NAME_HERE]`?

**Review by Klynger — APPROVED** [review]

**Review by klzns — APPROVED** [review]

**Pull request merged**